src/gui/mod.rs

```diff
@@ -1838,6 +1844,7 @@ impl std::fmt::Display for CatalogSource {
                 catalog::Source::Curse => "Curse",
                 catalog::Source::Tukui => "Tukui",
                 catalog::Source::WowI => "WowInterface",
+                catalog::Source::TownlongYak => "TownlongYak",
```

Comment:
Wondering if we should write: Townlong Yak

Reply:
Might as well put a `-` in at that point?

Reply:
Yeah lets do that.